protofsm/state_machine.go

```diff
@@ -595,9 +586,8 @@ func (s *StateMachine[Event, Env]) applyEvents(ctx context.Context,
 				//
 				//nolint:ll
 				err := fn.MapOptionZ(events.ExternalEvents, func(dEvents DaemonEventSet) error {
-					log.Debugf("FSM(%v): processing "+
-						"daemon %v daemon events",
-						s.cfg.Env.Name(), len(dEvents))
+					log.DebugS(ctx, "Processing daemon events",
```

Comment:
Is this upper case construct something new we're attempting to make standard with the new logging format?